src/main/java/frc/robot/RobotContainer.java

```diff
@@ -22,18 +23,25 @@
  * subsystems, commands, and button mappings) should be declared here.
  */
 public class RobotContainer {
+
+  public enum DriveTrainType {
+    TankDrive,
+    ArcadeDrive
+  }
+
   // The robot's subsystems and commands are defined here...
   private ChassisSubsystem chassisSubsystem_ = null;
   private ArmSubsystem armSubsystem_ = null;
   private DriverOI driverOI_ = null;
   private CoDriverOI coDriverOI_ = null;
-  private IntakeSubsystem intakeSubsystem = null;
+  private IntakeSubsystem intakeSubsystem_ = null;
```

Comment:
Java objects should not be initialized to null. In C you should always initialize variables when they are declared, in Java you almost never should initialize objects. Java initializes them to null by the standard.
```suggestion
  private IntakeSubsystem intakeSubsystem_;
```

Reply:
understood. habit carried from other languages.

Reply:
Actually, VSCode generated a constant variables and setting String to null. So, this isn't uncommon use.

Reply:
I am not sure how VSCode did that, it doesn't do it for me. 

Also, IntelliCode has an AI engine which is supposed to learn about the conventions in your code base. That might have something to do with it. Not a big deal either way.